lib/game_stats.rb

```diff
@@ -1,4 +1,35 @@
 
 
 module GameStats
+
+    def count_of_games_by_season 
+        games_per_season = Hash.new(0)
+        @games.each do |game|
+            if games_per_season.has_key?(game.season_id)
+                games_per_season[game.season_id] += 1
+            else
+                games_per_season[game.season_id] = 1
+            end
+        end
+        games_per_season
+    end
+
+    def average_goals_per_game
```

Comment:
I like how you set this up in an array so you can sum the elements and divide by the size in one go.

Reply:
Thanks Chee!